Makefile

```diff
@@ -17,7 +17,7 @@ WixDir	= $(BldDir)/wix
 Status	= $(BldDir)/status
 BinExtra= ssh #bash ls mount
 
-export PATH := /cygdrive/c/Program Files (x86)/WiX Toolset v3.10/bin:$(PATH)
+export PATH := $(WIX)/bin:$(PATH)
```

Comment:
Agree. But should we perhaps change this to be:

```
export PATH := $(shell cygpath -au "$$WIX")/bin:$(PATH)
```

Reply:
Sure! It works with windows path syntax, but conversion will definitely make it bulletproof.